### ISSUE TITLE
Hot key double press with super key

### ISSUE THREAD

**Issue by pszucs** [Ulauncher/Ulauncher#405]
When pressing the hotkey set for ulauncher (super + space), the space gets entered into the active applications window (regardless of whatever that is - browser, terminal, text editor, etc) and ulauncher doesn't open.

If I press the space again (still holding the super key) then ulauncher opens. I've tried other characters (for instance super + e), the issue is the same, the "e" is passed to the active application, ulauncher only opens when pressing the "e" again. Other key combinations not involving the super key work.

### Steps to reproduce
1. Press the hot key to open ulauncher

### Current behavior
When pressing the hot key ulauncher doesn't open. When pressing the second key of the key combination again (whilst holding the super key) ulauncher opens.

### Expected behavior
When pressing the hot key ulauncher should open without the second key of the hot key combo being passed to the active open window.

### Environment
**Ulauncher version**: 5.3.0~beta1 (tried it with 4.4 too, same issue)
**Linux distro**: elementary OS 5 Juno
**Desktop environment**: Pantheon

### Logs
https://gist.github.com/pszucs/b087145ea38237a0aa4c2eb53c06b53f

I set the hotkey to "super + e". In the log file on line 32 it shows the "e" being entered into the terminal window.

**Comment by gornostal** [comment]
You can't use `Super + Space` in Elementary because it's a hotkey for the default launcher. You probably know that, right?

I've tried Super+e and other hotkeys with Super, they have a problem as you described. I don't know how to fix that.

**Comment by pszucs** [comment]
I know it's the default hotkey, but it can be changed - I've done that, to make sure there's no conflict.

I've also tried changing the Super key behaviour in the settings ("Disabled", "Applications menu", "Shortcut overlay") but it didn't make any difference.

**Comment by devansh08** [comment]
Facing the same issue on Ubuntu 18.04. Tried with Ctrl+Space and Super+Space. Both times I had to keep Ctrl or Super pressed while hitting Space twice to trigger Ulauncher.

**Comment by pszucs** [comment]
> Facing the same issue on Ubuntu 18.04. Tried with Ctrl+Space and Super+Space. Both times I had to keep Ctrl or Super pressed while hitting Space twice to trigger Ulauncher.

That might be a separate issue as I can use Ctrl + Space, I only have issues with hotkeys using the Super key.

**Comment by endorama** [comment]
I can confirm this is happening in Elementary Juno, but I don't think the issue is in Ulauncher, feels more like the OS/DE "eating" the super keypress.

As a workaround, I'm successfully using `Ctrl+Super+space`.

**Comment by devansh08** [comment]
That's true. Even apps "eat" up the shortcuts. Example most IDEs have Ctrl+Space for auto complete.

**Comment by dolpa** [comment]
To fix it go to Setting ->  Mouse & Touchpad -> General  and try to disable "Reveal Point".
Worked for me.

**Comment by eliocamp** [comment]
Can confirm this on elementary OS 5.1.7 Hera. 

A workaround could be to add the `ulauncher` command to Super + Space as a custom keyboard. It works, but ulauncher launchs without focus and it also seems to take longer to appear.

**Comment by Dr-Chi** [comment]
I would love for this to be fixed also.  I even went into 'gsettings set org.gnome.desktop.wm.keybindings' to make sure Super+space wasn't doing something else.  As mentioned before Super+Space works, but only if you hit space twice.  It's like the Super key is being eaten on the first hit or something.

**Comment by bcolloran** [comment]
workaround for linux mint 20.1: basically, just make sure the mint menu isn't capturing `super` before anything else can. Steps:

- right click on the mint menu icon in your panel
- select "preferences" from the right click icon
- menu preferences > general, in the "menu button" section find "keyboard shortcut"
- there is no option to _disable_ the shortcut, but you can change it to some super obscture combination you won't need for anything else
- now go back to ulauncher: you should be able to capture `super` or `super+space` 
- I had to quit ulauncher and restart a couple times to get the capture to work the way I wanted, so if it doesn't seem to work immediately try that

**Comment by elogicuk** [comment]
I was running into the same issue on arch linux

super (win key) + space was taking 2 presses of the space bar to activate ulauncher

i fixed this by leaving ulauncher set to ctrl + space

and then making a custom keyboard shortcut that triggers ulauncher-toggle mapped to super + space

and its working great now
